src/HashingLogic.ts

```diff
@@ -69,6 +73,16 @@ export interface IRevocationLinks {
    * hash of type node attester is verifying
    */
   typeHash: string
+  /**
+   * RFC3339 timestamp of when the claim was issued
+   * https://tools.ietf.org/html/rfc3339
+   */
+  issuanceDate: string
+  /**
+   * RFC3339 timestamp of when the claim should expire
+   * https://tools.ietf.org/html/rfc3339
+   */
+  expirationDate: string
```

Comment:
These are just ISO-8601 right?

Reply:
it's iso-8601 + a couple more constraints. it's required in the verifiable claims spec draft from w3c